plugins/outputs/azure_monitor/README.md

```diff
@@ -40,6 +40,11 @@ written as a dimension on each Azure Monitor metric.
   ## The Azure Resource ID against which metric will be logged, e.g.
   ##   ex: resource_id = "/subscriptions/<subscription_id>/resourceGroups/<resource_group>/providers/Microsoft.Compute/virtualMachines/<vm_name>"
   # resource_id = ""
+  
+  ## Optionally, if in Azure US Government, China, or other sovereign
+  ## cloud environment, set the appropriate REST endpoint for receiving
+  ## metrics. (Note: region may be  unused in this context)
+  # endpoint_url = "https://monitoring.core.usgovcloudapi.net"
```

Comment:
Can you copy this to the SampleConfig() in azure_monitor.go too?  This will ensure that it is part of the generated config: `telegraf -usage azure_monitor`

Reply:
@danielnelson - I dropped that update into the `SampleConfig()` in commit c99192de